### DISCUSSION TITLE
Recommended videos, playlist subscription, supervised accounts

### DISCUSSION THREAD

**Discussion by dmlogv** [ViewTube/viewtube#1951] [Q&A]
Hi everything. Sorry for combining different questions, but they are coupled. 

I'm pretty new in ViewTube and I use it to serve my children wishes. It works perfectly.

But I'd like to see (or find out in the settings) the following features:

1. Show Recommended videos only from my subscriptions. Now it shows videos from unsubscribed channels too.
1. Allow to subscribe only to the specific playlist. There are channels with, em, different content, and I see it's a reasonable feature. (and https://github.com/ViewTube/viewtube/issues/74 of course). 
1. How about supervised accounts? I mean specific users that cannot search videos or subscribe to anything, but their supervisors can help them.

**Comment by moisout** [comment]
These are cool features! I'll keep them in mind.
